gnosis/safe/safe.py

```diff
@@ -75,7 +67,7 @@ class SafeInfo:
     version: str
 
 
-class SafeBase(ContractBase, metaclass=ABCMeta):
+class Safe(SafeCreator, ContractBase, metaclass=ABCMeta):
```

Comment:
Why Safe extends SafeCreator?

Reply:
To separate the logic of creatign Safes into another class. It's just for organization and maybe in the future to split Safe creation logic from Safe class